src/locales/strings/enUS.json

```diff
@@ -617,6 +617,12 @@
   "fio_get_fee_err_msg": "There was an issue getting fee. Please try later.",
   "fio_get_reg_info_err_msg": "There was an issue getting registration information.",
   "fio_register_address_err_msg": "There was an issue registering an address. Please try later.",
+  "send_fio_request_dialog_title": "Request by FIO Address",
+  "send_fio_request_error_addr_not_available": "The provided FIO address is not available",
+  "send_fio_request_error_memo_inline": "Memo must be less than 64 characters and contain symbols from latin alphabet",
```

Comment:
Latin alphabet is unclear. Use two error messages and see comment above.

Reply:
ok